packages/ts-docs-gen/src/api-items/definitions/api-accessor.ts

```diff
@@ -0,0 +1,58 @@
+import { Contracts } from "ts-extractor";
+import { GeneratorHelpers } from "../../generator-helpers";
+import { ApiDefinitionBase } from "../api-definition-base";
+import { ApiTypes } from "../api-type-list";
+
+export type ApiAccessorKinds = Contracts.ApiGetAccessorDto | Contracts.ApiSetAccessorDto;
+
+export class ApiAccessor extends ApiDefinitionBase<ApiAccessorKinds> {
+    private type: ApiTypes | undefined;
+
+    public get Type(): ApiTypes | undefined {
+        if (this.type != null) {
+            const resolvedType = this.resolveType();
+            if (resolvedType != null) {
+                this.type = GeneratorHelpers.SerializeApiType(this.ExtractedData, resolvedType);
+            }
+        }
+        return this.type;
+    }
+
+    private resolveType(): Contracts.ApiType | undefined {
+        let type: Contracts.ApiType | undefined;
+        if (this.Data.ApiKind === Contracts.ApiItemKinds.GetAccessor) {
+            // GetAccessor
+
+            type = this.Data.Type;
+        } else if (this.Data.Parameter != null) {
+            // SetAccessor
+
+            const apiParameter = this.ExtractedData.Registry[this.Data.Parameter.Ids[0]] as Contracts.ApiParameterDto;
+            if (apiParameter != null) {
+                type = apiParameter.Type;
+            }
+        }
+
+        return type;
+    }
+
+    public ToText(): string[] {
+        const name = this.Reference.Alias || this.Data.Name;
+        const abstract = this.Data.IsAbstract ? " abstract" : "";

```

Comment:
I need a dollar, dollar, dollar is what I need.

Reply:
Hey hey 🎵